### DISCUSSION TITLE
Ρόλοι στον οργανισμό

### DISCUSSION THREAD

**Discussion by stdi** [sdy60-2022/.github#28] [Ideas]
Θα θέλατε να δημιουργήσουμε παραπάνω ρόλους και πολιτικές αναβάθμισης σε αυτούς;

Θα μπορούσατε να προτείνετε ρόλους και κανόνες;

**Comment by gregtsag** [comment]
για τώρα πιστεύω καλά είμαστε έτσι, τουλάχιστον μέχρι να καταφέρουμε να βρούμε σε έναν ψιλοαντίστοιχο βαθμό όλοι τα πατήματα μας στην πλατφορμα. αν νιώσετε ότι σας έχουν πέσει πολλά εννοείται να το κουβεντιάσουμε. αυτό λέω εγώ ας πούνε και οι υπόλοιποι

**Comment by sotkot** [comment]
Μπορούμε να δούμε αυτό που προτείνεις Αλέξανδρε σε επόμενα tasks, ανάλογα με τις ανάγκες που θα προκύψουν.
